cab-token-generator/pom.xml

```diff
@@ -16,6 +16,33 @@
   <build>
     <sourceDirectory>java</sourceDirectory>
     <testSourceDirectory>javatests</testSourceDirectory>
+    <extensions>
+      <extension>
+        <!-- This extension is to fetch the OS label for the protoc plugin. -->
+        <groupId>kr.motd.maven</groupId>
+        <artifactId>os-maven-plugin</artifactId>
+        <version>1.7.1</version>
+      </extension>
+    </extensions>
+    <plugins>
+      <plugin>
+        <groupId>org.xolstice.maven.plugins</groupId>
+        <artifactId>protobuf-maven-plugin</artifactId>
+        <version>0.6.1</version>
+        <configuration>
+          <protoSourceRoot>${project.basedir}/proto</protoSourceRoot>
+          <protocArtifact>com.google.protobuf:protoc:${project.protobuf.version}:exe:${os.detected.classifier}</protocArtifact>
```

Comment:
Can we define a new maven property for the protoc version (i.e. something like `project.protoc.version`)? I would like to separate the Protoc and Protobuf-Java runtime to give us flexibility to move the runtime forward.

Reply:
Added